terraform/services.tf

```diff
@@ -126,8 +126,24 @@ module "trial_app_service_role" {
   ]
 }
 
+resource "azurerm_storage_account" "initstorageaccount" {
+  name                      = "sa${var.trial_name}init${var.environment}"
+  resource_group_name       = azurerm_resource_group.trial_rg.name
+  location                  = azurerm_resource_group.trial_rg.location
+  account_kind              = "StorageV2"
+  account_tier              = "Standard"
+  account_replication_type  = "LRS"
+  enable_https_traffic_only = true
+  allow_blob_public_access  = true
+}
+
+resource "azurerm_storage_share" "initstorageshare" {
+  name                 = "init"
+  storage_account_name = azurerm_storage_account.initstorageaccount.name
+  quota                = 1
+}
```

Comment:
Can this be defined inside the module and only pass a var if you want it or not?

Reply:
TODO in [ARTS-863](https://ndph-arts.atlassian.net/browse/ARTS-863)